docker/_include.sh

```diff
@@ -10,7 +10,12 @@ XDG_CACHE_HOME=${XDG_CACHE_HOME:-$HOME/.cache}
 PROJECT_NAME="commcarehq"
 
 function web_runner() {
-    sudo \
-        env DOCKER_DATA_HOME=$DOCKER_DATA_HOME XDG_CACHE_HOME=$XDG_CACHE_HOME\
-        docker-compose -f $DOCKER_DIR/compose/docker-compose-hq.yml -p $PROJECT_NAME $@
+    if [ `uname` == 'Linux' ]; then
```

Comment:
Since on the mac, docker is modifying a VM it has control over, it doesn't need sudo (and breaks if sudo is used).

http://ariya.ofilabs.com/2014/05/docker-on-os-x.html